src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -169,6 +169,10 @@
         .fl-preview-A {
             font-size: 2em;
         }
+        
+        .fl-preview-A:before {
+            content: "a";
```

Comment:
Hey Justin, this is needed for screenreaders to read it properly in browsers other than firefox. Otherwise all other browsers will still say the 'a'